src/main/java/com/couchbase/lite/Database.java

```diff
@@ -1142,6 +1145,39 @@ public synchronized void open() throws CouchbaseLiteException {
         open = true;
     }
 
+    /**
+     * Create a SymmetricKey object from the key (byte[32]) or password string.
+     * @param keyOrPassword
+     * @return
+     * @throws CouchbaseLiteException
+     */
+    @InterfaceAudience.Private
+    SymmetricKey createSymmetricKey(Object keyOrPassword) throws CouchbaseLiteException {
+        if (keyOrPassword == null)
+            return null;
+
+        byte[] rawKey = null;
+        if (keyOrPassword instanceof String) {
+            rawKey = ((EncryptableStore) store).derivePBKDF2SHA256Key(
+                    (String)keyOrPassword,
+                    DEFAULT_PBKDF2_KEY_SALT.getBytes(),
+                    DEFAULT_PBKDF2_KEY_ROUNDS);
+        } else {
```

Comment:
Do you suggested to changer? Personally I prefer the default convention from Android Studio so we will not forget to do it differently.